docs/release-process.md

```diff
@@ -40,13 +62,25 @@ Hotfixes are started off of master. Bugfixes off of stable.
 
```

Comment:
From line 61, should Bugfixes start off of develop?
```suggestion
Hotfixes are started off of master. Bugfixes off of develop.
```

Reply:
Hotfix is a fix against live production code. The master branch is always the latest available production source. 

A bugfix is a fix that can wait to be completed - it's not a high severity. So you treat it like a feature. It's not a feature thought because you're fixing an issue.